src/codegen/tests/code_gen_tests.rs

```diff
@@ -2234,7 +2234,7 @@ fn typed_enums_with_partly_initializers_are_generated() {
         TYPE MyEnum: BYTE(red := 7, yellow, green);
         END_TYPE
 
-        TYPE MyEnum: BYTE(a,b,c:=7,d,e,f:=twenty,g);
+        TYPE MyEnum2: BYTE(a,b,c:=7,d,e,f:=twenty,g);
```

Comment:
Changing the name back to `MyEnum` and then running this test's code results in 
```
thread 'main' panicked at src/resolver.rs:774:21:
new expression we did not visit yet
```

Reply:
I've removed the `panic!()` because it seemed to be too strict (imo) and all other resolver tests seem to work, A minimal reproducible example would be 
```
TYPE 
    foo: BYTE(a, b);
    foo: BYTE(c, d);
END_TYPE
```
where a statement annotation on `c` is missing, but this seems to happen only on duplicate enum types as far as I can tell. I think this was introduced for debugging purposes but was forgotten to be removed. Thoughts @ghaith ?